api-doc/api-doc.yaml

```diff
@@ -4181,7 +4168,7 @@ paths:
   /v1/programs/removeRolesInScope/{programId}:
```

Comment:
@MallanagoudaB I don't think add/remove api for role for solution and program are relavant now. please verify once

Reply:
@MallanagoudaB if not using we can deprecate it from api doc. else someone will refer it and try to use it